src/controllers/User-Sign-Up-Controller.coffee

```diff
@@ -93,5 +120,10 @@ class User_Sign_Up_Controller
   render_Page: (jade_Page,params)=>
     @.res.send @.jade_Service.render_Jade_File jade_Page, params
 
-
+  loadSecretFile:() ->
+    if (process.cwd().path_Combine('../../config/SiteData_TM/secrets.json').file_Exists())
```

Comment:
can you use __dirname instead of process.cwd()  it will be more robust for execution and unit tests